templates/noticeMessageReceiver.pug

```diff
@@ -0,0 +1,9 @@
+body
+  p Hello, #{message.toUserName}!

```

Comment:
This should be message.toUser.name.

... assuming that message has been retrieved and joined with the user info.